### ISSUE TITLE
Apply LogEventMapper to Crash LogEvents

### ISSUE THREAD

**Issue by laenger** [DataDog/dd-sdk-android#1022]
I would like to be able to modify each `LogEvent` generated by the crash reports feature.

I already use the log event mapper (`Configuration.setLogEventMapper(...)`) to modify `additionalProperties` of each regular `LogEvent`. That mapping, however, is not applied for logs generated as the result of a crash, as can be seen in the `PersistenceStrategy` implementation.

I would like to propose to apply the log event mapper to every `LogEvent`.

[CrashReportFilePersistenceStrategy.kt#L37](https://github.com/DataDog/dd-sdk-android/blob/1.13.0/dd-sdk-android/src/main/kotlin/com/datadog/android/error/internal/CrashReportFilePersistenceStrategy.kt#L37)
```kotlin
BatchFilePersistenceStrategy<LogEvent>(
    ...,
    LogEventSerializer(),
    ...
)
```

[LogFilePersistenceStrategy.kt#L43](https://github.com/DataDog/dd-sdk-android/blob/1.13.0/dd-sdk-android/src/main/kotlin/com/datadog/android/log/internal/domain/LogFilePersistenceStrategy.kt#L43)
```kotlin
BatchFilePersistenceStrategy<LogEvent>(
    ...,
    MapperSerializer(LogEventMapperWrapper(logEventMapper), LogEventSerializer()),
    ...
)
```

**Comment by laenger** [comment]
This may also solve #623

**Comment by xgouchet** [comment]
Hi @laenger, indeed this is a solution that could be done. We'll keep that in our backlog although this won't be our main priority right now, but since you already identified the actions to take, feel free to send a PR our ways with relevant unit tests added.

**Comment by 0xnm** [comment]
This is available since SDK 2.0, closing the issue.
